### ISSUE TITLE
🆕 Software Suggestion | Mattermost

### ISSUE THREAD

**Issue by Mikaela** [privacytools/privacytools.io#1066]
## Basic Information

**Name:** Mattermost
**Category:** Team chat (#1065)
**URL:** https://mattermost.com/

## Description

> Mattermost is an open-source, self-hostable online chat service. It is designed as an internal chat for organisations and companies, and mostly markets itself as an alternative to Slack.[6][7] 

* https://en.wikipedia.org/wiki/Mattermost

Warning: Mattermost doesn't yet have E2EE.

* [Feature request for OTR](https://mattermost.uservoice.com/forums/306457-general/suggestions/12818799-off-the-record-messaging)
    * [Ticket for OTRv2](https://mattermost.atlassian.net/browse/MM-669)
* [Feature request for end to end encryption support](https://mattermost.uservoice.com/forums/306457-general/suggestions/36662833-end-to-end-e2e-encryption-support)

**Comment by goapunk** [comment]
Apart from lacking E2EE the community edition comes without any form of access control / policy rights. Any user can archive a channel (which then is gone from the channel list and excluded from search results) and only admins can recover it. Also any user can remove other users from channels and change channel names and topics. (https://github.com/mattermost/mattermost-server/issues/6320)

**Comment by Mikaela** [comment]
Thanks for the note, I don't think Mattermost can be listed due to that restriction.

**Comment by goapunk** [comment]
Looks like they started on fixing the archived channel stuff, so things might not be as bad anymore.

**Comment by Oymate** [comment]
Open?

**Comment by freddy-m** [comment]
@Oymate are you asking for this issue to be re-opened?

**Comment by Oymate** [comment]
Yeah, the  main issue seems to be solved and I do think it's a pretty good suggestion(better than element anyway)
